standard/info/commons/info.lua

```diff
@@ -24,7 +25,12 @@ return {
 			},
 		},
 	},
-	defaultGame = 'commons',
-
+	config = {
+		squad = {
+			hasPosition = false,
```

Comment:
Wasn't in the PR yet, was waiting for #4158

Reply:
It can be merged as is, but the combination of this PR and another would allow for the deletion of additional customs with help of that flag. Should have made it clearer

Reply:
Now it's in this PR